django/text_extractor/utils.py

```diff
@@ -285,3 +285,12 @@ def create_searchable_pdf(input_file, add_header):
         output.add_page(new_pdf_page.pages[0])
 
     return output, all_text
+
+
+def shorten_input_name(input_name, max_length=35):
```

Comment:
You may as well just always use a UUID, then you never have to try to make filenames unique, right?